src/3_PB/VerbaliInterni/VerbaleInterno_240329/content.typ

```diff
@@ -0,0 +1,40 @@
+#import "meta.typ": inizio_incontro, fine_incontro, luogo_incontro
+#import "functions.typ": glossary, team
+#let participants = csv("participants.csv")
+
+= Partecipanti
+/ Inizio incontro: #inizio_incontro
+/ Fine incontro: #fine_incontro
+/ Luogo incontro: #luogo_incontro
+
+#table(
+  columns: (3fr, 1fr),
+  [*Nome*], [*Durata presenza*],
+  ..participants.flatten()
+)
+= Sintesi Elaborazione Incontro
+
+== Conclusione _Manuale Utente v1.0_
+
+L'incontro si è aperto discutendo dello stato di completamemnto del _Manuale Utente_, che è necessario terminare e revisionare interamente entro la fine dello sprint corrente; la stesura del documento può dirsi terminata, se non che occorre aggiungere qualche dettaglio implementativo in più:
+
+- Nella sezione riguardante l'installazione del prodotto software, occorre specificare la procedura da seguire per ottenere ed impostare:
+  - API key del servizio utilizzato per simulare percorsi realistici per le biciclette elettriche;
+  - API key del webhook Discord utilizzato per impostare un canale testuale come ente di ricezione delle notifiche che denotano il superamento di soglie predeterminate sui dati simulati;
+- Aggiungere una breve sezione riguardante la componente di allarmi e relative notifiche menzionata sopra;
+- Corredare le descrizioni testuali di screenshot che aiutino l'utente a comprendere meglio la funzionalità in esame.
+Matteo Rango si occupa del primo punto, mentre Nancy Kalaj si occupa degli ultimi due punti.
+
+== Conclusione _Piano di Qualifica v2.0_
+
+Anche il _Piano di Qualifica_ deve essere terminato inserendo i dati pertinenti allo sprint corrente al termine dello stesso e aggiornando i grafici di conseguenza; occorre, inoltre, fare una breve analisi dell'andamento delle metriche negli sprint 9-15 (periodo pre-PB), confrontandolo anche con l'andamento nel periodo precedente, e riportarla sotto i grafici (se ne occupa Riccardo Costantin). Infine, occorre verificare che le modifiche apportate in precedenza all'_Analisi dei Requisiti_ siano di fatto riportate anche all'interno dei test di accettazione e di sistema, ove ve ne è la necessità; dell'eventuale aggiornamento dei test si occupa Nancy Kalaj.
```

Comment:
qui c'è da stare attenti perchè va bene che noi l'analisi delle metriche la facciamo solo adesso ma tecnicamente dovrebbe essere fatta a ogni sprint per valutare le azione correttiva e robe del genere quindi non ci butterei l'attenzione.

Reply:
inoltre il cambiamento dei test dovrebbe avvenire i contemporanea con la modifica di adr, (transizione), qunadì anche qui eviterei di buttarci troppo l'attenzione.

Reply:
1) mi sto riferendo al paragrafetto di testo che aggiungiamo sotto i grafici in vista della PB, mi sembra abbastanza scontato che lo scriviamo effettivamente solo prima della PB avendo tutti i dati a disposizione; non credo questo implichi che non li abbiamo mai considerati prima

2) mmh magari cambio leggermente il fraseggio per far intendere che le cose sono state fatte in contemporanea (che è vero)

Reply:
per il primo si effettivamente, ci sta. 
per il secondo perfetto.